requirements.txt

```diff
@@ -1,2 +1,2 @@
 requests
-elasticsearch==7.14.2
+elasticsearch==8.14.0
```

Comment:
According depend a bot there is a 8.16.x https://github.com/openmsa/python-sdk/pull/104

Reply:
@Vignaudo is was mostly to avoid major changes if any  on latest version :D , but maybe it's the right occasion to move directly to elasticsearch python latest version  and adapt if necessary ? @antoine-brun @vmonnier  ?

Reply:
Humm it's a minor version change regarding your PR. The major change is 7.x to 8.x

Reply:
this major change is required to have this https://github.com/ubiqube/dcim/pull/20 merged. 

We have "cascade"  dependencies here 

With MSA 3.0.0 and Python version 

- https://github.com/ubiqube/dcim project requires pandas and pandas is not available "natively"  in this new msa python version, so latest version should added 
- pandas requires latest version of NumPy
-   NumPy requires **at least** elasticsearch-py **8.14.0**

Reply:
can we take care of this in v3.1.0 ?
3.0.0 is meant to be for demo that we control and none of these demo are dependent  on ES py API
a WF can always call the ES API with some curl calls